ls1intum/Artemis
# `Adaptive learning`: Fix score display for competency details view

### Checklist
#### General


#### Server

### Motivation and Context



Resolves #7600 

### Description


### Steps for Testing

Prerequisites:
- 1 Student
- 1 Instructor
- 1 Programming Exercise with a submission. The deadline has to be in the future.

1. Log in as a Student
2. Go to "Competencies" 
3. Now you should see the competency with the exercise, navigate to it
4. On the right side above the Due Date timer, it should show your score.
5. Wait until the deadline has passed (or set it in the past)
6. Go to "Competencies"
7. Now you should see the competency with the exercise, navigate to it
8. On the right side above the Due Date timer, it should show your score.

### Testserver States
> [!NOTE]
> These badges show the state of the test servers.
> Green = Currently available, Red = Currently locked

![](https://byob.yarr.is/ls1intum/Artemis/artemis-test1)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test2)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test3)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test4)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test5)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test6)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test9)

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests

### Test Coverage







### Screenshots
